tck/old-tck/source/src/com/sun/ts/tests/jsf/spec/el/elresolvers/TestServlet.java

```diff
@@ -47,7 +49,6 @@
 import jakarta.servlet.ServletException;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-import jakarta.servlet.jsp.JspFactory;
 import jakarta.servlet.jsp.PageContext;
```

Comment:
All jakarta.servlet.jsp imports can be removed since their usage has also been removed.